src/registry_schemas/schemas/party.json

```diff
@@ -0,0 +1,113 @@
+    "type": "object",
+    "title": "The Corp Parties Schema",
+    "definitions": {
+        "person": {
+            "type": "object",
+            "properties": {
+                "partyType": {
+                    "type": "string",
+                    "title": "The type of party",
+                    "enum": [
+                        "org",
+                        "person"
+                    ]
+                },
+                "firstName": {
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "lastName": {
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "middleInitial": {
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "orgName": {
+                    "type": "string",
+                    "maxLength": 50
+                },
+                "orgType": {
+                    "type": "string",
+                    "maxLength": 50
+                }
+            },
+            "anyOf": [
+                {
+                  "properties": {
+                    "partyType": { "const": "person" }
+                  },
+                  "required": ["firstName", "lastName"]
+                },
+                {
+                  "properties": {
+                    "partyType": { "const": "org" }
+                  },
+                  "required": ["orgName"]
+                }
+              ],
+            "required": [
+                "partyType"
+            ]
+        },
+        "party": {
+            "type": "object",
+            "properties": {
+                "person": {
+                    "$ref": "#/definitions/person"
+                },
+                "deliveryAddress": {
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
+                },
+                "mailingAddress": {
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
+                },
+                "title": {
+                    "type": "string"
+                },
+                "appointmentDate": {
+                    "type": "string",
+                    "format": "date"
+                },
+                "cessationDate": {
+                    "type": [
+                        "string",
+                        "null"
+                    ],
+                    "format": "date"
+                },
+                "roles": {
+                    "type": "array",
+                    "items": {
+                        "enum": [
+                            "Director",
+                            "Incorporator"
+                        ]
+                    }
+                }
+            },
+            "required": [
+                "person",
+                "roles",
+                "mailingAddress",
+                "deliveryAddress",
+                "appointmentDate"
```

Comment:
This is currently passed from the front end. Is this a requirement as well? The schema says yes..

Reply:
yes it is required from the front end